helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -405,6 +413,11 @@ void checkConnected(long timeout) {
 
   void addListener(Object listener, PropertyKey propertyKey, ChangeType changeType,
       EventType[] eventType) {
+    addListener(listener, propertyKey, changeType, eventType, -1);
+  }
+
+  void addListener(Object listener, PropertyKey propertyKey, ChangeType changeType,
+      EventType[] eventType, long periodicRefreshTriggerInterval) {
```

Comment:
Refresh and Trigger are duplicates somehow.
I think we just call it periodicTriggerInterval for generic.